draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1088,42 +1143,50 @@ optical network domain; and
 - one TE link is reported for each OMS link and, optionally, for
 each OTN link.
 
-Since the MDSC delegates optical path computation to its underlay O-PNCs, the following information can be abstracted and not reported at
+Since the MDSC delegates optical path computation to its underlay O-PNCs,
+the following information can be abstracted and not reported at
 the MPI:
 
 - the optical parameters required for optical path computation, such
-as those detailed in {{?I-D.ietf-ccamp-optical-impairment-topology-yang}};
+as those detailed in
+{{?I-D.ietf-ccamp-optical-impairment-topology-yang}};
 
 - the underlay OTS links and ILAs of OMS links;
 
 - the physical connectivity between the optical transponders and the
 ROADMs.
 
 The OTN Topology Model also reports the CBR client LTPs that
-terminates the cross-technology Ethernet links: once CBR client LTP is reported for
+terminates the cross-technology Ethernet links: once CBR client LTP is
+reported for
 each CBR or multi-function client interface on the optical nodes (see
-sections 4.4 and 5.1 of {{?I-D.ietf-ccamp-transport-nbi-app-statement}} for the description of multi-function
+sections 4.4 and 5.1 of {{?I-D.ietf-ccamp-transport-nbi-app-statement}}
+for the description of multi-function
 client interfaces).
 
 The Ethernet Topology Model reports the Ethernet client LTPs that
-terminate the cross-technology Ethernet links: one Ethernet client LTP is reported
+terminate the cross-technology Ethernet links: one Ethernet client LTP is
+reported
 for each Ethernet or multi-function client interface on the optical
 nodes.
 
 The optical transponders and, optionally, the OTN access cards, are
 abstracted at MPI by the O-PNC as Trail Termination Points (TTPs),
 defined in {{!RFC8795}}, within the optical network topology. This
 abstraction is valid independently of the fact that optical
-transponders are physically integrated within the same WDM node or
-are physically located on a device external to the WDM node since it
-both cases the optical transponders and the WDM node are under the
-control of the same O-PNC and abstracted as a single WDM TE Node at the O-MPI.
+transponders are physically integrated within the same DWDM node or
+are physically located on a device external to the DWDM node since it
+both cases the optical transponders and the DWDM node are under the
+control of the same O-PNC and abstracted as a single DWDM TE Node at the
```

Comment:
I-D.ietf-ccamp-optical-impairment-topology-yang defines the 'WDM TE node' (and not the 'DWDM TE node')

```suggestion
control of the same O-PNC and abstracted as a single WDM TE Node at the
```